manifests/charts/istio-control/istio-discovery/templates/istiod-injector-configmap.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
 data:
 {{/* Scope the values to just top level fields used in the template, to reduce the size. */}}
   values: |-
-{{ pick .Values "global" "istio_cni" "sidecarInjectorWebhook" "revision" | toPrettyJson | indent 4 }}
+{{ pick .Values "global" "istio_cni" "sidecarInjectorWebhook" "revision" "meshConfig" | toPrettyJson | indent 4 }}
```

Comment:
Why is this change needed?

Reply:
it's needed because they reference mesh config from the injection template. the injector passes them directly so we shouldn't do this, in a follow-up we can change this to unblock others